docker/Dockerfile-dhvirtualenv

```diff
@@ -61,6 +61,8 @@ RUN apt-get update -qq -o Acquire::Languages=none \
         python3-setuptools \
         python3-venv \
         sqlite3 \
+        libsqlite3-dev \
```

Comment:
this is not the right thing to be doing. We don't need `libsqlite3-dev` to build the package. I added it, in error, to the debian build control file in #5675, but it shouldn't be there, and if it's not there, it's not needed here. Can you remove it instead?

Reply:
Oh I see, that's why it complained about it during the build then?

Sure, I can remove it.